app/Core/Database.php

```diff
@@ -29,7 +29,10 @@ public static function getConnectionData () {
 	 * @return bool database status flag
 	 */
 	public static function initialize () : bool {
-		self::$database = new \mysqli ('localhost:3306', 'root', 'glitch123');
+		$config_data = parse_ini_file ("../app/server.ini");
+
+		self::$database = new \mysqli ($config_data['host'].':'.$config_data['port'], $config_data['user'],
```

Comment:
When you have to wrap a parameter list, do it with each parameter on a separate line.

Reply:
done and noted